### PR TITLE
feat: add caip adapters generation script

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

Does what it says on the box - avoids having to cd into `caip` to generate caip adapters.
Regenerates adapters while at it.

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

- contributes to https://github.com/shapeshift/web/issues/6383

Low - ensure assets list still looks sane

> What protocols, transaction types or contract interactions might be affected by this PR?

## Testing

- Assets still look sane

### Engineering

- ^

### Operations

- ^

### CODE REVIEW

**Comment by gomesalexandre** [comment]
* **#6573** <a href="https://app.graphite.dev/github/pr/shapeshift/web/6573?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gomesalexandre and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: package.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
     "dev:packages": "yarn tsc --build --watch --preserveWatchOutput tsconfig.packages.json",
     "tenderly": "yarn tenderly:spawn && yarn env e2e && yarn concurrently --kill-others -n \"rpc-proxy,unchained-proxy\" \"yarn tenderly:rpc-proxy\" \"yarn tenderly:unchained-proxy\"",
     "env": "yarn ts-node --skipProject ./scripts/bootstrap-env.ts",
+    "generate:caip-adapters": "npm --prefix ./packages/caip run generate",
```
Comment (gomesalexandre) [inline package.json:28]: Not a mistake, tried many many hoops to get this running with `yarn` with no success 😥

Comment (kaladinlight) [inline package.json:28]:
```suggestion
    "generate:caip-adapters": "yarn workspace @shapeshiftoss/caip generate",
```

Comment (kaladinlight) [reply package.json:28]: below suggestion should work by leveraging yarn workspace directive

**Review by kaladinlight — APPROVED** [review]

**Pull request merged**